tests/acceptance/test_files/000.basics.bats

```diff
@@ -6,6 +6,16 @@ load "$LIB_BATS_SUPPORT/load.bash"
     assert_success
 }
 
+@test "steampipe plugin" {
```

Comment:
update the name to something like `steampipe plugin help is displayed when no sub command given`